src/pages/Projects.jsx

```diff
@@ -8,7 +8,11 @@ const Projects = () => {
     // either make Hero reusable somehow (and move it to components/global)
     // or use <Box component="img" />
     <>
-      <Hero />
+      <Hero
```

Comment:
I find this photo to be pretty dark - testing on all of them I think we should change ALL the gradient opacities to be `0.3` vs the `0.5` we have.

Reply:
should still have plenty of contrast. If not we can figure it out at the meeting.

Reply:
I changed to 0.3 for now I think it looks good!